libs/platform/user/kernel_um.h

```diff
@@ -35,10 +35,188 @@ extern "C"
 #define STATUS_NOT_IMPLEMENTED ((NTSTATUS)0xC0000002L)
 #define STATUS_NOT_FOUND ((NTSTATUS)0xC0000225L)
 #define STATUS_UNSUCCESSFUL ((NTSTATUS)0xC0000001L)
+#define STATUS_BUFFER_TOO_SMALL ((NTSTATUS)0xC0000023L)
 #define PASSIVE_LEVEL 0
 #define DISPATCH_LEVEL 2
 
     // Typedefs
+
+    typedef struct _SE_EXPORTS
+    {
+
+        //
```

Comment:
This comment style differs from the one normally used in this project. (a single '//' comment indicator immediately followed by the comment body)

Reply:
fixed.